lang/test/org/partiql/lang/eval/visitors/SelectStarVisitorTransformTests.kt

```diff
@@ -1,12 +1,16 @@
-package org.partiql.lang.ast.passes
+package org.partiql.lang.eval.visitors
 
 import junitparams.Parameters
 import org.junit.Test
+import org.partiql.lang.ast.passes.GroupByItemAliasRewriter
+import org.partiql.lang.ast.passes.PipelinedRewriter
+import org.partiql.lang.ast.passes.RewriterTestBase
+import org.partiql.lang.ast.passes.RewriterTransformBridge
 
-class SelectStarRewriterTests : RewriterTestBase() {
+class SelectStarVisitorTransformTests : RewriterTestBase() {
 
-    // SelectStar rewriter depends on UniqueNameMetas injected by GroupByItemAliasRewriter.
-    private val rewriters = PipelinedRewriter(GroupByItemAliasRewriter(), SelectStarRewriter())
+    // SelectStar visitor transform depends on UniqueNameMetas injected by GroupByItemAliasVisitorTransform.
+    private val rewriters = PipelinedRewriter(GroupByItemAliasRewriter(), RewriterTransformBridge(SelectStarVisitorTransform(), ion))
```

Comment:
After `GroupByItemAliasRewriter`'s PR (#304) is merged into the `visitor-transform` branch along with a VisitorTransform version of `PipelinedRewriter`, will update these tests to use `TransformTestCase` rather than `RewriterTestCase`.

Reply:
Cool.

Reply:
Have you done this yet?

Reply:
Not yet, but now that this PR is approved, I can finish up the `PipelinedRewriter` migration [PR](https://github.com/partiql/partiql-lang-kotlin/pull/320), which will update these tests and remove other rewriter references.